src/CSI.IBTA.Administrator/Clients/UserServiceClient.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Net;
+using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Text;
+using CSI.IBTA.Administrator.Constants;
+using CSI.IBTA.Administrator.Endpoints;
+using CSI.IBTA.Administrator.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using CSI.IBTA.Shared.DTOs.Errors;
+using CSI.IBTA.Shared.DTOs.Login;
+using CSI.IBTA.Shared.Entities;
+using CSI.IBTA.Shared.Types;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CSI.IBTA.Administrator.Clients
+{
+    internal class UserServiceClient : IUserServiceClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILogger<AuthClient> _logger;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly IJwtTokenService _jwtTokenService;
+
+        public UserServiceClient(HttpClient httpClient, IJwtTokenService jwtTokenService, ILogger<AuthClient> logger, IConfiguration configuration, IHttpContextAccessor httpContextAccessor)
+        {
+            _logger = logger;
+            _httpClient = httpClient;
+            _jwtTokenService = jwtTokenService;
+            var userServiceApiUrl = configuration.GetValue<string>("UserServiceApiUrl");
+            if (string.IsNullOrEmpty(userServiceApiUrl))
+            {
+                _logger.LogError("UserServiceApiUrl is missing in appsettings.json");
+                throw new InvalidOperationException("AuthServiceApiUrl is missing in appsettings.json");
+            }
+            _httpClient.BaseAddress = new Uri(userServiceApiUrl);
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        public async Task<List<Employer>?> GetEmployers(string token)
+        {
+            if (_httpContextAccessor.HttpContext == null)
```

Comment:
This comment is related to the classes EmployerClient, EmployerUserClient and UserServiceClient, I just do not want to repeat it 3 times. 

 _httpContextAccessor is not using and could be removed. 

Token receiving logic is repeatable. Lets consider reusing it with inheritance or any other approach you design.